### ISSUE TITLE
Read the user preferred language

### ISSUE THREAD

**Issue by kelvinst** [railsinstaller/website#35]
The gem [http_accept_language](https://github.com/iain/http_accept_language) (already added to Gemfile) has an API to read the user preferred language from the browser.
This feature was already implemented in old pull request #31, but, for some reason, it doesn't work properly in production. It was reverted and corrected by @emachnic in #32 and #33, respectively. But in the corrections, this feature was crashed.
Basically the change we need is in [application controller](https://github.com/kelvinst/railsinstaller.org/blob/translation_refactoring/app/controllers/application_controller.rb) of #31. But we need to make sure that will work on production.
